quinn-proto/src/congestion.rs

```diff
@@ -1,27 +1,67 @@
 //! Logic for controlling the rate at which data is sent
 
-use std::time::{Duration, Instant};
+use crate::connection::paths::RttEstimator;
+use std::fmt::Debug;
+use std::time::Instant;
 
+mod bbr;
+mod bbr_bw_estimation;
+mod bbr_min_max;
 mod cubic;
 mod new_reno;
 
+pub use bbr::{BBRConfig, BBR};
 pub use cubic::{Cubic, CubicConfig};
 pub use new_reno::{NewReno, NewRenoConfig};
 
 /// Common interface for different congestion controllers
-pub trait Controller: Send {
+pub trait Controller: Send + Debug {
+    /// Packets were sent, called when the send buffer is sent, not when each packet is sent.
```

Comment:
Would this still be in line with the intended semantics?
```suggestion
    /// One or more packets were just sent
```